apps/studio/src-commercial/entrypoints/renderer.ts

```diff
@@ -155,6 +156,11 @@ import { ForeignCacheTabulatorModule } from '@/plugins/ForeignCacheTabulatorModu
     const handler = new AppEventHandler(app)
     handler.registerCallbacks()
     await store.dispatch('initRootStates')
+    try {
+      await new WebPluginManager().initialize();
+    } catch (e) {
+      log.error("Error initializing web plugin manager", e)
```

Comment:
does this need to do more than log the error?

Reply:
@azmy60 outstanding q here

Reply:
I'll try to fix

Reply:
@rathboma Is adding a noty enough?

Reply:
done a71dedff196e59a93813058e286bc671c8745d53